easybuild/easyconfigs/s/SeaView/SeaView-5.0.5-GCCcore-11.2.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'Tarball'
+
+name = 'SeaView'
+version = '5.0.5'
+
+homepage = 'https://doua.prabi.fr/software/seaview'
+description = """SeaView is a multiplatform, graphical user interface for multiple sequence alignment
+ and molecular phylogeny."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://doua.prabi.fr/software/seaview_data']
+sources = [{'download_filename': '%(namelower)s%(version_major)s-64.tgz', 'filename': '%(name)s-%(version)s-64.tgz'}]
+checksums = ['e8d844ee5d41f7e5626631ac1eb75086add6a24bfe8a5aa69d045c622a8abd8d']
```

and going to source would also allow for a reliable download link, since binary tarball would change for any bugfix release as well
```suggestion
source_urls = ['ftp://pbil.univ-lyon1.fr/pub/mol_phylogeny/seaview/archive/']
sources = ['%(namelower)s_%(version)s.tar.gz']
checksums = [whateveritmightbe]
```